subscription_manager.go

```diff
@@ -725,7 +719,7 @@ func processSubscribePayload(m *SubscriptionManager, payload subscribeMessage) {
 	}
 }
 
-func createPNFilesEvent(filePayload interface{}, m *SubscriptionManager, actualCh, subscribedCh, channel, subscriptionMatch, issuingClientID string, userMetadata interface{}, timetoken int64, messageType MessageType, spaceId SpaceId) *PNFilesEvent {
+func createPNFilesEvent(filePayload interface{}, m *SubscriptionManager, actualCh, subscribedCh, channel, subscriptionMatch, issuingClientID string, userMetadata interface{}, timetoken int64, typ string, spaceId SpaceId) *PNFilesEvent {
```

Comment:
Is that a typo?
```suggestion
func createPNFilesEvent(filePayload interface{}, m *SubscriptionManager, actualCh, subscribedCh, channel, subscriptionMatch, issuingClientID string, userMetadata interface{}, timetoken int64, type string, spaceId SpaceId) *PNFilesEvent {
```

Reply:
Nope. I had plenty of options to choose from, but `type` is not one of them. It's a reserved keyword so we need to be creative with words :)